src/Microsoft.Extensions.Caching.Memory/project.json

```diff
@@ -23,7 +23,11 @@
     "Microsoft.Extensions.Options": "1.0.0-*"
   },
   "frameworks": {
-    "net451": {},
+    "net451": {
```

Comment:
Ah ok. So should this target `netstandard1.2` as this can work for net451 and also should work for net46 as it supports `netstandard1.3`, which is a super-set?